src/inputs/Table/components/ExportButton/ExportButton.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import DownloadIcon from '@mui/icons-material/Download';
+import { IconButton, Tooltip } from '@mui/material';
+import PropTypes from 'prop-types';
+
+export const ExportButton = (props) => {
+  const { onExport, isReady, label } = props;
+  return (
+    <Tooltip title={label}>
+      <span>
+        <IconButton onClick={onExport} size="small" color="primary" disabled={!isReady} data-cy="export-button">
+          <DownloadIcon fontSize="inherit" />
+        </IconButton>
+      </span>
+    </Tooltip>
+  );
+};
+
+ExportButton.propTypes = {
+  /*
+   * Function use to export a file from Table
+   */
+  onExport: PropTypes.func,
+  /*
+   * Boolean value to know if there is some data fully loaded
+   */
+  isReady: PropTypes.bool.isRequired,
```

Comment:
Could we rename this prop to `disabled`? (and set its value to the value of `!isReady` in its parent component)